src/utilities/updateBrowserUrl.browser.spec.ts

```diff
@@ -0,0 +1,35 @@
+
+import { expect, test, vi } from 'vitest'
+import { updateBrowserUrl } from '@/utilities/updateBrowserUrl'
+
+test('updates the window location', () => {
+  vi.spyOn(window.location, 'assign').mockImplementation(() => {})
```

Comment:
these `mockImplementation` calls make sure the actual location isn't updated. I tried a few different attempts to keep the location changes isolated to each test but this was the only thing I tried that worked.